exec/java-exec/src/main/resources/rest/profile/profile.ftl

```diff
@@ -129,10 +130,10 @@
                 </tr>
               </thead>
               <tbody>
-                <#list model.getOptionList() as option>
+                <#list options?keys as name>
```

Comment:
Neither this nor the original design displays the keys as sorted. Sorted display is a fairly traditional to save the user from a random search for each value. Since we are changing the behavior, should we fix the sorting issue as well?